lib/git_mock.ex

```diff
@@ -52,4 +58,33 @@ defmodule Gitea.GitMock do
     repo_name = Gitea.Helpers.get_repo_name_from_url(repo_path <> ".git")
     {:ok, "To ssh://gitea-server.fly.dev:10022/myorg/#{repo_name}.git\n"}
   end
+
+  @doc """
+  `commit/2` (mock) simulate a commit and always returns {:ok, commit_info}
+
+    ## Examples
+    iex> GitMock.commit("my-repo", ["any", "args"])
+    {:ok, "[master dc5b0d4] test msg\n Author: Al Ex <c@t.co>\n 1 file changed, 1 insertion(+)\n"}
+  """
+  @spec commit(Git.Repository.t(), [any]) :: {:ok, any}
```

Comment:
need these for https://github.com/dwyl/gitea-demo/pull/5#issuecomment-1138454750 
ref: #28